server/src/lib.rs

```diff
@@ -73,13 +73,17 @@ pub fn is_email_regex(test: &str) -> bool {
   EMAIL_REGEX.is_match(test)
 }
 
-pub fn is_image_content_type(test: &str) -> bool {
-  match isahc::get(test) {
-    Ok(res) => match res.headers().get("Content-Type") {
-      Some(header) => header.to_str().unwrap_or("not_an_img").contains("image"),
-      None => false,
-    },
-    Err(_) => false,
+pub fn is_image_content_type(test: &str) -> Result<(), failure::Error> {
+  if isahc::get(test)?
+    .headers()
+    .get("Content-Type")
+    .ok_or_else(|| format_err!("No Content-Type header"))?
+    .to_str()?
+    .starts_with("image")
```

Comment:
Better use `image/`? Also the if condition is very long, I would put that into a variable.

Reply:
I'll change to `image/`, but I don't wanna use a var cause we aren't using its value anyway.

Reply:
I meant putting the bool in a var before using it in the if, but its not much of a difference.